pkg/batcher/createfleet.go

```diff
@@ -52,6 +52,7 @@ func execCreateFleetBatch(ec2api ec2iface.EC2API) BatchExecutor[ec2.CreateFleetI
 	return func(ctx context.Context, inputs []*ec2.CreateFleetInput) []Result[ec2.CreateFleetOutput] {
 		results := make([]Result[ec2.CreateFleetOutput], 0, len(inputs))
 		firstInput := inputs[0]
+		logging.FromContext(ctx).Infof("Batched %d items together for createFleet", len(inputs))
```

Comment:
I'm not convinced we want to info log on all of these.

Reply:
Agreed. Staging environment changes. This will be removed when the final PR is raised